yarn.config.cjs

```diff
@@ -110,12 +116,14 @@ function enforceConsistentDependenciesAcrossTheProject({Yarn}) {
 /**
  *  Enforces each package having the same Embrace metadata
  */
-function enforceEmbraceMetadata({ Yarn }) {
+function enforceEmbraceMetadata({Yarn}) {
   for (const workspace of Yarn.workspaces()) {
-    if (workspace.manifest.private) continue;
+    if (workspace.manifest.private) {
+      continue;
+    }
 
     workspace.set("embrace", {
-      iosVersion: "6.4.2",
+      iosVersion: "6.5.1",
```

Comment:
this is the line I changed manually for bumping the version, the rest are only linter changes (done by the IDE when I pressed command + s)